docs/ru/easytofly.md

```diff
@@ -0,0 +1,142 @@
+
+Результатом является облако точек, которое можно использовать при дальнейшей работе с пакетами ROS, позволяющими производить действия с этой структурой данных (такие как: перевод в другие структуры данных, построение карты, передача данных на другое оборудование).
+
+Для установки датчика на Clover 4 было разработано и подготовлено для печати на 3D принтере специальное крепление, которое для экономии высоты было совмещено с креплением для аккумулятора.
+
+Далее предстояла разработка программного обеспечения, которое выполняло все преобразования, как типов данных, так и геометрии (соглашения о системах координат полетного контроллера и ROS отличаются). После этого обработанные данные направляются в полетный контроллер, который отвечает за принятие решения: остановить летящий БПЛА или продолжать полет.
+
+#### Аппаратное решение из 4-х лазерных дальномеров
+
+Помимо лидара, было принято решение использовать статические дальномеры для удешевления системы. Нами было разработано аппаратное решение на базе Arduino Nano и дальнедистанционных датчиков времени пролета VL53L1X. Изделие совместимо с разработанным нами креплением.
+
+Далее было написано программное обеспечение для работы с оборудованием. Было необходимо передать данные с микроконтроллера, установленного на Arduino Nano на микрокомпьютер, представить показания с датчиков в том же виде, в котором они представлены при получении данных с лидара. И после этого произвести те же действия, что и с лидаром.
+
+### Программная подсистема «Мастер первоначальной настройки БПЛА»
+
+Нами была запланирована разработка еще одной программной подсистемы, которая позволит настроить БПЛА, используя единый пользовательский веб-интерфейс, который «проведет» пользователя через весь процесс настройки. Также в данный веб-интерфейс должен быть встроен монитор текущего состояния БПЛА и редактор настроек.
+
+В первую очередь нами был сделан каркас будущего интерфейса.
+
+Также на данный момент реализован интерфейс калибровки датчиков БПЛА со встроенной анимацией для удобной работы с ним. Работу интерфейса калибровки можно увидеть на видео.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/7pV9BsczxMY" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Аппаратная составляющая АПК
+
+Также перед нами стояла задача разработать защищенный корпус БПЛА, с возможностью быстрой установки используемого оборудования, который не будет готов к полету и обучающимся не будет необходимости его собирать. На данный момент готова конструкторская документация изделия, а само изделие проходит апробацию.
+
+## Использование нашего продукта
+
+Репозиторий проекта доступен по ссылке https://github.com/easy-to-fly/easy-to-fly.
+
+### Сборка крепления для датчиков
+
+Вам понадобится:
+- **4x** стойки M3x6
+- **1x** крепаление для батареи (ССЫЛКА)
```

Comment:
The link is missing.

Reply:
https://github.com/CopterExpress/clover/pull/309/commits/bce1b0960f7b7430141087de4afcf4068fac00d9